### ISSUE TITLE
Unable to view and download files in public share view link

### ISSUE THREAD

**Issue by candideu** [nocodb/nocodb#3541]
**Project Info**
```
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.96.1**
```

**Steps To Reproduce**

1. Use the latest version of NocoDB.
2. Go to a table.
3. Add an attachment field and add audio files.
4. Click on "Share View".
5. Go to the share view link in incognito mode or in on a different browser.
6. Error: The attachment column will be blank. It won't allow viewers to click on or download the file.

**Expected Behavior**

In the last version I used (0.92.3), the attachments were visible in the public shared view link. This feature is really important since I'm hosting a public database where people can download audio files.

**Screenshots**

Before:

![image](https://user-images.githubusercontent.com/55474996/188905683-f6546310-c8cc-456a-99ec-96e33e6a0775.png)

Currently:

![image](https://user-images.githubusercontent.com/55474996/188905298-04dbc5df-44a1-4ccd-ab29-b617a044d389.png)

**Comment by o1lab** [comment]
Ok thanks for reporting. Are you behind a reverse proxy ?

**Comment by candideu** [comment]
> Ok thanks for reporting. Are you behind a reverse proxy ?

I'll be honest: I'm not sure what that is... I've hosted it on Railway.app and connected my domain.

**Comment by o1lab** [comment]
Ok that helps - will see how to go about it.

**Comment by o1lab** [comment]
Fixed in 0.96.2

**Comment by candideu** [comment]
Incredible! Thank you for the quick fix
